### PR TITLE
[ADD] add module res_users_case_insensitive

### PR DESCRIPTION
**Pull request by houssine78**

### CODE REVIEW

File: res_users_case_insensitive/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2015-TODAY LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Case Insensitive Logins",
+    "summary": "Makes the login field of res.users case insensitive.",
+    "version": "9.0.1.0.0",
```
Comment (remytms) [inline res_users_case_insensitive/__manifest__.py]: Version number should be : 11.0.1.0

File: res_users_case_insensitive/models/res_users.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api, fields
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+    login = fields.Char(
+        'Login',
+        size=64,
```
Comment (remytms) [inline res_users_case_insensitive/models/res_users.py]: I don't see what's the benefit of limiting the size of the login to 64 characters. In odoo 11, this field is not limited. See this definition in the `odoo/addons/base/res/res_users.py` file :

```python
    login = fields.Char(required=True, help="Used to log into the system")
```

**Review by remytms — CHANGES_REQUESTED** [review]
I've done my comment. At least change the version number. :)

File: res_users_case_insensitive/__manifest__.py (hunk 1 above)
Comment (remytms) [reply res_users_case_insensitive/__manifest__.py]: In fact, following [semver](https://semver.org/), it should be : 11.0.1.0.0

**Comment by houssine78** [comment]
@remytms done

**Pull request merged**